src/main/java/fi/otavanopisto/kuntaapi/server/integrations/tpt/TptConsts.java

```diff
@@ -0,0 +1,21 @@
+package fi.otavanopisto.kuntaapi.server.integrations.tpt;
+
+/**
+ * Constants for te-palvelut.fi -integration
+ * 
+ * @author Antti Leppä
+ */
+public class TptConsts {
+  
+  public static final String IDENTIFIER_NAME = "TPT";
+  public static final String ORGANIZATION_SETTING_AREA = "tpt.area";
+  public static final String SYSTEM_SETTING_BASE_URL = "tpt.baseUrl";
+  public static final String DEFAULT_BASE_URL = "https://paikat.te-palvelut.fi";
+  public static final String AREA_SEARCH_PATH = "/tpt-api/tyopaikat?alueet=%s";
+  public static final String JOB_LINK_PATH = "/tpt/%d";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)